specs/001-si-richiede-di/contracts/instruction.md

```
@@ -0,0 +1,62 @@
+## Public API
+
+### Instruction
+```rust
+pub struct Instruction {
+    mnemonic: String,
+    operands: Vec<Operand>,
+}
+
+impl Instruction {
+    pub fn new(mnemonic: &str) -> Self
+    pub fn add_operand(&mut self, operand: Operand)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)